recipes/ale/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "ale" %}
```

Comment:
The name `ale` is used in other package managers by a different package https://repology.org/project/ale-anti-lamenessing-engine/versions